assets/controller.yaml

```diff
@@ -41,17 +41,29 @@ spec:
                 secretKeyRef:
                   name: ebs-cloud-credentials
                   key: aws_access_key_id
+                  optional: true
```

Comment:
now that the Secret holding the credentials will always hold the `credentials` field, is there any point in keeping these parameters around vs just relying on AWS_CONFIG_FILE?

Reply:
hmm, i don't want to depend on the new migration unless we have it backported to 4.6. so maybe we can remove it when we have that finished?

Reply:
:+1:  One of these days openshift/cloud-credential-operator#268 will merge into 4.6